docs/system_overview/troubleshooting.md

```diff
@@ -780,6 +780,9 @@ If the server *has been* restarted after running `iexit`, the server will stand
 
 If your users are authenticating via PAM (e.g. `pam_password` scheme) and find that they are being made to re-authenticate after only a few minutes, this section should provide an explanation and a way to remedy the situation.
 
+!!! Note
+    This situation is normally accompanied by a high occurrence of `CAT_PASSWORD_EXPIRED` in the server log.
```

Comment:
Not sure if this deserves a `!!! Note` box, but I don't feel too strongly about that.

Reply:
That's fair. Will adjust.

Reply:
Done.